recipes/libx11/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+import subprocess
+import shlex
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+def remove_prefix(s, prefix):
+    return s[len(prefix):] if s.startswith(prefix) else s
+
+
+class LibX11Conan(ConanFile):
+    name = "libx11"
+    license = "X11"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.x.org/wiki/"
+    description = "Client interface to the X Window System, otherwise known as \'Xlib\'",
+    settings = "os", "compiler", "build_type", "arch"
+    _required_system_package = "libx11-dev"
```

Comment:
would be much better to use pkg-config to check package availability, so it would be completely distro agnostic